api/src/client/WebserviceLicenseStatusProcessorClient.test.ts

```diff
@@ -1,10 +1,10 @@
-import { LicenseStatusResults } from "@api/types";
```

Comment:
This commit only contains changes to the imports, I temporary separated them so you can focus on the next commit